src/screens/TransactionScreen/TransactionScreen.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import { Header, Text, Paper, TransactionIcon, Icon, Button, Token } from '@components';
+import { BasicLayout } from '@layouts';
+import { useLanguage, useTransaction } from '@hooks';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { RootStackParamList } from '@src/routes/types.routes';
+import { useState } from '@hookstate/core';
+import { globalWalletState } from '@stores/WalletStore';
+import { networks } from '@models/network';
+import { Snackbar } from 'react-native-paper';
+import * as Clipboard from 'expo-clipboard';
+import * as Haptics from 'expo-haptics';
+import { smallWalletAddress } from '@models/wallet';
+import { TokenType } from '@styles';
+import { truncate } from '@src/hooks/useTransaction';
+import styles from './TransactionScreen.styles';
+
+type Props = NativeStackScreenProps<RootStackParamList, 'TransactionScreen'>;
+const TransactionScreen = ({ route }: Props) => {
+	const { transaction } = route.params;
+	const { i18n } = useLanguage();
+	const { network } = useState(globalWalletState()).value;
```

Comment:
I think we could update the NetworkContext to bring this "network", so we can call it using the "useNetwork()" hook instead of invoking "useState" and "globalWalletState()" here.

Reply:
useNetwork() brings this network object but somehow its not working.